doc/03_research/01_acting/03_acting_testing.md

```diff
@@ -0,0 +1,24 @@
+# How to test acting components
```

Comment:
As you already mentioned, I think it is time to open up a new folder for these kinds of documents.

Reply:
Fixed in 21267cb6dd92f8ee5dc715489ecf5cfd0f278ef0